content/en/agent/guide/linux-agent-2022-key-rotation.md

```diff
@@ -100,7 +100,12 @@ $ sudo rpm --import /tmp/DATADOG_RPM_KEY_FD4BF915
 {{< tabs >}}
 {{% tab "Debian/Ubuntu" %}}
 
-If the file `/usr/share/keyrings/datadog-archive-keyring.gpg` exists, the new key is trusted and no further action is needed. This file is created either by an [installation method](#install-methods-that-automatically-trust-the-new-gpg-key) or by installing the [`datadog-signing-keys` package](#the-datadog-signing-keys-package).
+A host correctly trusts the new key either one of these conditions is true:
+
+- The file `/usr/share/keyrings/datadog-archive-keyring.gpg` exists and the Datadog source list file (usually `/etc/apt/sources.list.d/datadog.list`) contains the option `[signed-by=/usr/share/keyrings/datadog-archive-keyring.gpg]`.
+- The Datadog source list file doesn't contain the `signed-by` option, but `datadog-signing-keys` version 1.1.0 or later is installed, which results in presence of `/etc/apt/trusted.gpg.d/datadog-archive-keyring.gpg` file.
```

Comment:
Not a blocker for this PR, but could we later add a script here that can check if either of these conditions is true, and outputs `New GPG key is trusted` (+ `0` exit code) or `New GPG key is NOT trusted` (+ non-`0` exit code). Unless this seems completely useless based on the customer feedback we've already received.

Reply:
So far it doesn't seem necessary, but if the kind of feedback that we're getting changes, I can easily do that.